roofit/roostats/inc/RooStats/DebuggingSampler.h

```diff
@@ -46,7 +46,7 @@ namespace RooStats {
 
       // Main interface to get a ConfInterval, pure virtual
      virtual SamplingDistribution* GetSamplingDistribution(RooArgSet& paramsOfInterest)  {
-       paramsOfInterest = paramsOfInterest; // avoid warning
+       static_cast<void>(paramsOfInterest); // avoid compiler warning
```

Comment:
`(void)paramsOfInterest`, please :-)

Reply:
Didnt work here I got ``` /home/oksana/CERN_sources/root/builds/include/RooStats/DebuggingSampler.h:49:13: error: variable has incomplete type 'void'
       void(paramsOfInterest); // avoid warning
            ^
```
with static_cast it work..

Reply:
rather than
```
void(paramsOfInterest); // avoid warning
```
you meant
```
(void) paramsOfInterest; // avoid warning
```
cheers.